doc/sphinx-guides/source/installation/config.rst

```diff
@@ -2352,6 +2352,15 @@ when using it to configure your core name!
 
 Can also be set via *MicroProfile Config API* sources, e.g. the environment variable ``DATAVERSE_SOLR_PATH``.
 
+dataverse.concurrency.max-async-indexes
```

Comment:
The setting name is not consistent with the code.

Reply:
I hope I got it right this time: ```dataverse.solr.concurrency.max-async-indexes```